test/spec/modules/lemmaBidAdapter_spec.js

```diff
@@ -161,6 +161,7 @@ describe('lemmaBidAdapter', function() {
         expect(data.site.publisher.id).to.equal(bidRequests[0].params.pubId.toString()); // publisher Id
         expect(data.imp[0].tagid).to.equal('1'); // tagid
         expect(data.imp[0].bidfloorcur).to.equal(bidRequests[0].params.currency);
+        expect(data.imp[0].bidfloor).to.equal(bidRequests[0].params.floor);
```

Comment:
You should add tests for the changes you made.  
This added line is not testing your changes, here you are only comparing `undefined` with `undefined`.  

Run `gulp test-coverage` and `gulp view-coverage` to see your coverage.

Reply:
I have added test coverage for getFloor function changes.